qiskit_experiments/library/characterization/fine_amplitude.py

```diff
@@ -261,19 +273,66 @@ def _default_experiment_options(cls) -> Options:
 
         Experiment Options:
             gate (Gate): FineSXAmplitude calibrates an SXGate.
-            add_sx (bool): This option is False by default when calibrating gates with a target
-                angle per gate of :math:`\pi/2` as it is not necessary in this case.
-            add_xp_circuit (bool): This option is False by default when calibrating gates with
-                a target angle per gate of :math:`\pi/2`.
+            add_cal_circuits (bool): If set to True then two circuits to calibrate 0 and 1 points
+                will be added. This option is set to False by default for ``FineSXAmplitude``
+                since the amplitude calibration can be achieved with two SX gates and this is
+                included in the repetitions.
             repetitions (List[int]): By default the repetitions take on odd numbers for
                 :math:`\pi/2` target angles as this ideally prepares states on the equator of
                 the Bloch sphere. Note that the repetitions include two repetitions which
                 plays the same role as including a circuit with an X gate.
         """
         options = super()._default_experiment_options()
         options.gate = SXGate()
-        options.add_sx = False
-        options.add_xp_circuit = False
+        options.add_cal_circuits = False
         options.repetitions = [0, 1, 2, 3, 5, 7, 9, 11, 13, 15, 17, 21, 23, 25]
 
         return options
+
+
+class FineZXAmplitude(FineAmplitude):
+    r"""A fine amplitude experiment for the :code:`RZXGate(np.pi / 2)`.
+
+    # section: overview
+
+        :class:`FineZXAmplitude` is a subclass of :class:`FineAmplitude` and is used to set
+        the appropriate values for the default options to calibrate a :code:`RZXGate(np.pi / 2)`.
+    """
+
+    def __init__(self, qubits: Sequence[int], backend: Optional[Backend] = None):
+        """Initialize the experiment."""
+
+        # We cannot use RZXGate since it has a parameter so we redefine the gate.
+        # Failing to do so causes issues with QuantumCircuit.calibrations.
+        gate = Gate("rzx", 2, [], label="Rzx(pi/2)")
```

Comment:
Then we can give another name to this to avoid name conflict such as "szx"? Or probably you can link parameter directly to pulse amplitude via instmap. You also need to set explicit basis gate? It would be great if you could write `# section: example` about how to set custom ZX gate.

Reply:
Good points. Thanks. See here: [1a00617](https://github.com/Qiskit/qiskit-experiments/pull/655/commits/1a00617c73ac2c0d05b62e3a1c14463c47457efd)